app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -21,7 +21,8 @@ def build_request_body
             account_last_name: applicant[:last_name],
             account_telephone: '', # applicant[:phone], decision was made not to send phone
             account_drivers_license_number: applicant[:state_id_number].gsub(/\W/, ''),
-            account_drivers_license_issuer: applicant[:state_id_jurisdiction].to_s.strip,
+            account_drivers_license_type: "us_dl",
+            account_drivers_license_issuer: applicant[:state_id_jurisdiction] || applicant[:state],
```

Comment:
it's technically possible for users to change their address from their state id to a different one... I would recommend not falling back just so we don't accidentally move a state ID number across state lines

```suggestion
            account_drivers_license_issuer: applicant[:state_id_jurisdiction],
```

Reply:
In that case we might want to ensure we are actually capturing `state_id_jurisdiction` from licenses. I'm not seeing it in the outbound JSON.